src/main/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepository.java

```diff
@@ -207,16 +207,16 @@ private Query getQueryByParameters(MetaData metadata, final Map<String, String>
     }
 
     private Query getCountQueryByMetaData(MetaData metadata) {
-        return queryBuilderFactory.count(em)
-                                  .whereMetadata(metadata)
-                                  .build();
+        return queryBuilderFactory.count(em, metadata)
+            .whereMetadata(metadata)
+            .build();
     }
 
     private Query getQueryByMetaData(MetaData metadata) {
-        return queryBuilderFactory.select(em)
-                                  .whereMetadata(metadata)
-                                  .orderByCreatedDate(metadata.getSortDirection().orElse("asc"))
-                                  .build();
+        return queryBuilderFactory.select(em, metadata)
+            .whereMetadata(metadata)
+            .orderByCreatedDate(metadata.getSortDirection().orElse("asc"))
+            .build();
```

Comment:
The 2 lines above should be indented and aligned with `.select`

Reply:
indented